### DISCUSSION TITLE
Jupyter notebook with illustrative example

### DISCUSSION THREAD

**Discussion by b2jia** [ggirelli/iFISH-probe-design#21] [Ideas]
Thank you for writing and providing this great tool!

I'm familiar with traditional bioinformatics sequencing tools but this is my first time working with FISH. I see the ```Examples``` tab on the documentation is still in development. I'm happy to help with this, with a little bit of guidance. As a new user, I'd love to see a ```.ipynb``` notebook that strives to reproduce the results of a published paper. This will motivate the caveats that arise when designing probes for a real experiment. @ggirelli what do you think? Would you have a paper in mind?

**Comment by ggirelli** [comment]
Hi @b2jia ! Thanks for showing interest in the iFISH pipeline, and sorry for the late reply.
It really is a great idea to have a Jupyter python notebook with a few examples!

I will soon comment with a few real-life cases (and associated papers) that we might want to include.
I just set up a branch (**dev-notebook-docs**) where we could work on the notebook.
Or feel free to start already by forking and sending a pull request. 😄
